language-support/ts/daml-ledger/BUILD.bazel

```diff
@@ -6,22 +6,20 @@ load("@build_bazel_rules_nodejs//:index.bzl", "pkg_npm")
 load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
 load("//language-support/ts:defs.bzl", "da_ts_library")
 load("//language-support/ts:eslint.bzl", "eslint_test")
+load("//language-support/ts:jest.bzl", "jest_test")
 load("//language-support/ts:typedoc.bzl", "ts_docs")
 load("@build_environment//:configuration.bzl", "sdk_version")
 
 da_ts_library(
     name = "daml-ledger",
-    srcs = ["index.ts"],
+    srcs = glob([
+        "**/*.ts",
+    ]),
     module_name = "@daml/ledger",
     visibility = ["//visibility:public"],
     deps = [
         "//language-support/ts/daml-types",
-        "@language_support_ts_deps//@mojotech/json-type-validation",
-        "@language_support_ts_deps//@types/ws",
-        "@language_support_ts_deps//cross-fetch",
-        "@language_support_ts_deps//events",
-        "@language_support_ts_deps//isomorphic-ws",
-        "@language_support_ts_deps//ws",
+        "@language_support_ts_deps//:node_modules",
```

Comment:
Why do you need to depend on all of `node_modules`?

Reply:
That's what we do in `@daml/react` as well. I pretty much copied from there. I'll try to make it more restrictive here.

Reply:
Keeping the old list and only adding `@language_support_ts_deps//@types/jest` does the job too. Doing that now.

Reply:
Thanks!